src/v2.ts

```diff
@@ -64,37 +64,26 @@ export function wrapV2<T>(
   return (cloudEvent: CloudEvent) => cloudFunction.run(cloudEvent);
 }
 
-/** @return Helper type that suggests useful fields to override */
-export type CloudEventOverrides = {
-  source?: string;
-  subject?: string;
-  type?: string;
-  data?: any;
-  params?: Record<string, string>;
-};
-
 /**
  * @param cloudFunction Populates default values of the CloudEvent
- * @param {CloudEventOverrides} cloudEventOverride Used to override CloudEvent params.
+ * @param {Partial<CloudEvent>} cloudEventOverride Used to override CloudEvent params.
```

Comment:
You need to create a new `DeepPartial` or `RecursivePartial` type: [(explanation)](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBASwHY2FAZgQwMbDgYQBsIBXAEwFEA3YFAHgBUA+OAbwFgAoOOTMBAGpoAzgghIAXHABEARgB0ABmkAaLj2GkouKcJhRkAcwDc6uMJIAjAFbBsMAPy79R09zhlMMTFIZuAvlxcoJCwiChoWLhwAApWAMpWACJemGxmALbAwsKYhsBSHO48nt7OBkgmZjzWmpJwJEgA1kgQAO5IbjyB7l4uliSowk5wAEp20GR0ehWGKuYulUwBQZzY4npwYJiwCJiEUjE7MHuEdESklDT0cZaJlineTCwAvOnupT7vPDwA9L9wAByEFQcBgAAs8OsMvxCGg4GgoNA4JCoMB5HAAEJ2TAkYR4I67fZ0FgIYRIADk8DR2BIUFENDUxTg-0Q8CymCQwjgAE9SHAMnj4GAkVQEGQ8GiAI4kBBoshwdAIYCEMjciDocxWAC0WRyeWy1QF2Vy+UKRpqdXNzJ+KJVxCk0ja0FVqgtcH8TNtnrMPs4PWC4Gg8BgPLAeCSwGAYEJJ2JDARIFQSDVcAgNjsMBYRR4AG0ANLAHnhOBNIsauAMAC6IwYBaLVcTydT6ds9jgDjgkejsdOjHrPKrLF8A6rXADaw28AlPeOpyk3Zjc+JF3I1FoMDot3uj0wzzgbxzHlS1ttrOBbV5-OwnLg0NheAs2FwOXQJEIhGLlhxeIjUaXRKEIg3I0nSDLAEaeqmgU3y2nAtTiKecE8JCH4QI6zpQK6XpwX63o4X6-hAA)